### DISCUSSION TITLE
Dark mode ( GitHub Mobile )

### DISCUSSION THREAD

**Discussion by emrocode** [community/community#42345] [Mobile]
Does this only happen to me? or is it a bug (dark mode is not displaying correctly) 🤔

It seems that it is already solved 😎

![Screenshot_20221221_222253.jpg](https://user-images.githubusercontent.com/92260849/209034452-a11cb682-1151-461c-9bba-4e221c536114.jpg)

**Comment by MitkoVtori** [comment]
I think it's a bug only for you.
You can search your problem [here](https://github.com/contact) or contact the support through [email](mailto:support@github.com)

**Reply by MitkoVtori** [reply]
@emrocode By the way, I noticed something. You have exactly as much repositories as the picture shows. And the stars of each repo are right for you. And with the 54 total stars, it's not really your stars, it's how many repos you liked.

**See, if you go to your account and then click on stars:**

![Screenshot 2022-12-22 082839](https://user-images.githubusercontent.com/112943652/209071519-3dfe242e-a20f-4f81-8e3f-a4aabaaa233e.png)

**You'll see, that's NOT your repos, but repos you've like:**

![Screenshot 2022-12-22 082919](https://user-images.githubusercontent.com/112943652/209071602-df84813e-0e09-49e7-b992-e8df71f4d92b.png)

So it's not a bug, just missunderstanding :)

**Reply by mahmoudsharaf101** [reply]
know if you have any questions or comments on this page 📄

**Comment by MitkoVtori** [comment]
@emrocode I noticed something. You have exactly as much repositories as the picture shows. And the stars of each repo are right for you. And with the 54 total stars, it's not really your stars, it's how many repos you liked.

**See, if you go to your account and then click on stars:**

![Screenshot 2022-12-22 082839](https://user-images.githubusercontent.com/112943652/209071519-3dfe242e-a20f-4f81-8e3f-a4aabaaa233e.png)

**You'll see, that's NOT your repos, but repos you've like:**

![Screenshot 2022-12-22 082919](https://user-images.githubusercontent.com/112943652/209071602-df84813e-0e09-49e7-b992-e8df71f4d92b.png)

So it's not a bug, just missunderstanding :)
